packages/@aws-cdk/aws-eks/test/integ.eks-cluster-handlers-vpc.ts

```diff
@@ -0,0 +1,25 @@
+/// !cdk-integ pragma:ignore-assets
+import { App } from '@aws-cdk/core';
+import * as eks from '../lib';
+import { TestStack } from './util';
+
+const CLUSTER_VERSION = eks.KubernetesVersion.V1_18;
+
+
+class EksAllHandlersInVpcStack extends TestStack {
```

Comment:
Reluctantly, adding another integ test because this property affects the cluster deployment and operation.